third_party/move/move-compiler-v2/tests/ability-checker/ability_violation.move

```diff
@@ -0,0 +1,28 @@
+module 0x42::ability {
+	// fun move_from_no_key<T>(addr: address) {
```

Comment:
Remove?

Reply:
You can try to uncomment that function, and it will result in a panic
`panicked at 'expected Type::Struct, found: TypeParameter(0)', third_party/move/move-model/src/ty.rs:536:13`
I am not sure why this is happening, but I suspect it's a bug.

Reply:
Yes: 
(1) Please add a note here that uncommenting this code causes panic. In this way, we know you kept this commented out code for a reason, instead of someone just removing commented out code as useless code.
(2) File an issue to investigate this problem, so that we don't forget about it.

Reply:
https://github.com/aptos-labs/aptos-core/issues/11377